micromamba/src/run.cpp

```diff
@@ -274,7 +274,7 @@ set_run_command(CLI::App* subcom)
 #ifndef _WIN32
     static std::string specific_process_name;
     subcom->add_option(
-        "--pname",
+        "--label",
         specific_process_name,
         "Specifies the name of the process. If not set, a unique name enerated by deriving from the executable name will be generated.");
```

Comment:
I think the sentence also needs a bit of love. `enerated` is no word I recognize :)

Reply:
woups, will improve that XD